app/models/user.rb

```diff
@@ -1,5 +1,4 @@
 class User < ApplicationRecord
-
   devise :database_authenticatable, :registerable,
```

Comment:
I think we need a line with ```belongs_to :family``` here

Reply:
Above line 2

Reply:
We had that but took it out. But we need to create a user without having a family for now. Because we haven't created any families.

Reply:
Ok, I understand

Reply:
You can still have it but mark it as optional